cfgov/jinja2/v1/_includes/on-demand/secondary-navigation.html

```diff
@@ -1,9 +0,0 @@
-{# ==========================================================================
-
-   Render secondary navigation, for use in the global_include used
```

Comment:
I think this removal is fine. For reference, the secondary nav is this on e.g. https://www.consumerfinance.gov/about-us/careers/current-openings/

![screen shot 2018-07-09 at 3 14 10 pm](https://user-images.githubusercontent.com/704760/42471166-bf1a3f6c-838a-11e8-8e90-7fa5b66dd03a.png)